plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/fs/nio/EFSNIOFolder.java

```diff
@@ -49,7 +55,8 @@ public void create(boolean force, boolean local, IProgressMonitor monitor) throw
 
     public void create(int updateFlags, boolean local, IProgressMonitor monitor) throws CoreException {
         try {
-            Files.createDirectory(getNioPath());
+            Path result = Files.createFile(getNioPath().resolve(EMPTY_FOLDER_FAKE_FILE_NAME));
```

Comment:
You cannot do it for all file systems. We need it only for GCP

Reply:
Then I need this class to not be final

Reply:
I think it is better to use some flag in FS provider (DBFVirtualFileSystem? or maybe additional interface)